addons/website/views/website_templates.xml

```diff
@@ -594,33 +591,22 @@
     </xpath>
 </template>
 
-<template id="language_selector" name="Language Selector">
-    <ul class="js_language_selector mb0 list-inline" t-if="(request and request.is_frontend_multilang and len(languages) &gt; 1) or (website and (editable or translatable))">
-        <li class="list-inline-item">
-            <div class="dropup">
-                <button class="btn btn-sm btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                    <span t-esc="list(filter(lambda lg : lg[0] == lang, languages))[0][2].split('/').pop()"/>
-                    <span class="caret ml4"/>
-                </button>
-                <div class="dropdown-menu" role="menu">
-                    <t t-foreach="languages" t-as="lg">
-                        <a t-att-href="url_for(request.httprequest.path + '?' + keep_query(), lang_code=lg[0])"
-                           class="dropdown-item js_change_lang"
-                           t-att-data-url_code="lg[1]">
-                            <t t-esc="lg[2].split('/').pop()"/>
-                        </a>
-                    </t>
-                </div>
-            </div>
-        </li>
+<template id="language_selector" inherit_id="portal.language_selector">
+    <!-- Always show the language selector on editable websites -->
+    <xpath expr="//*[hasclass('js_language_selector')]" position="attributes">
+        <attribute name="t-if" add="(website and (editable or translatable))" separator=" or "/>
```

Comment:
We should use a t-set to avoid future breaking change
```
<t t-set='needShow' t-value='(request and request.is_frontend_multilang and len(languages) &gt; 1)'/>
<ul class="js_language_selector mb0 list-inline" t-if="need_show">
```

```
<xpath expr=//t-set position="after">
   <t t-set='needShow' t-value='needShow or  (website and (editable or translatable))'/>
</xpath>
```

Reply:
Yep indeed, bad rush, my bad